src/calculateRentalCost.js

```diff
@@ -29,13 +29,16 @@
 
 function calculateRentalCost(days) {
   let cost = 0;
+  const BASE_PRICE = 40;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
 
   if (days >= 3 && days < 7) {
```

Comment:
You can do the same with days:
const LONG_TERM = 7;
const SHORT_TERM = 3;

Reply:
Agreed, you should use consts for these durations.